.gitignore

```diff
@@ -13,4 +13,5 @@ DerivedData/
 *.mode2v3
 *.perspectivev3
 *.moved-aside
-
+CocoaPods\ Install/Pods/
+CocoaPods\ Install/Podfile.lock
```

Comment:
I think the `Podfile.lock` file should stay in git, as this is the one which guarantee we're using the same version. Just the `Pods` folder should be gone. This is the same with `bundler`.

Reply:
when running pod install on client side, Podfile.lock will be created automatically

Reply:
Yes I know, but it may contain different versions

Reply:
OK

Reply:
You need to commit the .lock file again, and make sure it's the last version :0

Reply:
YES

Reply:
The Podfile.lock is missing bugsnag ^^

Reply:
fixed